cfe_internal/enterprise/federation/federation.cf

```diff
@@ -566,6 +599,10 @@ bundle agent entry
       "CFEngine Enterprise Federation Configuration"
         handle => "config",
         usebundle => config;
+    am_off::
+      "CFEngine Enterprise Federation Transport Off"
+        handle => "clean_when_off",
+        usebundle => clean_when_off;
     enabled::
       "CFEngine Enterprise Federation Transport User"
```

Comment:
This doesn't work down here. "enabled" class is still defined when "am_off" is defined so all operations will still take place.

@nickanderson I will re-work these "enabled" and "am_off" classes tonight so they actually do what is needed.

Reply:
I think I have this fixed.